pkg/security/module/server_linux.go

```diff
@@ -192,7 +192,11 @@ func (a *APIServer) GetStatus(_ context.Context, _ *api.GetStatusParams) (*api.S
 		apiStatus.SelfTests = a.selfTester.GetStatus()
 	}
 	apiStatus.PoliciesStatus = a.policiesStatus
-	apiStatus.SECLVariables = a.GetSECLVariables()
+
+	seclVariables := a.GetSECLVariables()
+	for _, seclVariable := range seclVariables {
+		apiStatus.SECLVariables = append(apiStatus.SECLVariables, seclVariable)
+	}
```

Comment:
Can't we just make an append ? Like:
	`	apiStatus.SECLVariables = append(apiStatus.SECLVariables, seclVariables...)`

Same on `server_windows.go`

Reply:
`seclVariables` is a map (not exactly sure why though)

Reply:
oh ok, I though it was a slice